java/mtsj/core/pom.xml

```diff
@@ -365,7 +378,7 @@
         <mts.database.profile>hana</mts.database.profile>
         <mts.database.groupId>com.sap.cloud.db.jdbc</mts.database.groupId>
         <mts.database.artifactId>ngdbc</mts.database.artifactId>
-        <mts.database.version>2.2.16</mts.database.version>
+        <mts.database.version>2.3.48</mts.database.version>
```

Comment:
version seems to be redundant as the dependency was not using this variable.

Reply:
will be taken care after merge